Libplanet/Libplanet.csproj

```diff
@@ -67,12 +66,9 @@ https://docs.libplanet.io/</Description>
     <PackageReference Include="BouncyCastle.NetCore" Version="1.8.6" />
     <PackageReference Include="Caching.dll" Version="1.4.0.1" />
     <PackageReference Include="Destructurama.Attributed" Version="2.0.0" />
-    <PackageReference Include="Equals.Fody" Version="4.0.1" />
-    <PackageReference Include="Fody" Version="6.1.1">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
```

Comment:
These dependency removals should be recorded in the changelog.  Note that there is section *Dependencies*.